exporter/sumologicexporter/fields.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sumologicexporter
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+)
+
+// fields represents metadata
+type fields map[string]string
+
+// asString returns fields as ordered key=value string with `, ` as separator
+func (f fields) asString() string {
+	rv := make([]string, 0, len(f))
+	for k, v := range f {
+		rv = append(rv, fmt.Sprintf("%s=%s", k, v))
+	}
+	sort.Strings(rv)
+
+	return strings.Join(rv, ", ")
+}
+
+// format converts sourceFormat to the string.
+// it replaces %{key} with the appropriate value
```

Comment:
```suggestion
// format converts sourceFormat to string.
// It replaces %{key} with the appropriate value
```

What is "appropriate"? It's not obvious

Reply:
`corresponding`